stable/traefik/templates/configmap.yaml

```diff
@@ -100,18 +105,21 @@ data:
       {{- if .Values.kubernetes.ingressClass }}
     ingressClass = {{ .Values.kubernetes.ingressClass | quote }}
       {{- end}}
-      {{- if .Values.kubernetes.ingressEndpoint }}
-      [kubernetes.ingressEndpoint]
-        {{- if .Values.kubernetes.ingressEndpoint.hostname }}
-      hostname = {{ .Values.kubernetes.ingressEndpoint.hostname | quote }}
-        {{- end}}
-        {{- if .Values.kubernetes.ingressEndpoint.ip }}
-      ip = {{ .Values.kubernetes.ingressEndpoint.ip | quote }}
-        {{- end}}
-        {{- if .Values.kubernetes.ingressEndpoint.publishedService }}
-      publishedService = {{ .Values.kubernetes.ingressEndpoint.publishedService | quote }}
-        {{- end}}
+    {{- end}}
+    {{- if .Values.kubernetes.ingressEndpoint }}
+    [kubernetes.ingressEndpoint]
+      {{- if .Values.kubernetes.ingressEndpoint.hostname }}
+    hostname = {{ .Values.kubernetes.ingressEndpoint.hostname | quote }}
       {{- end}}
+      {{- if .Values.kubernetes.ingressEndpoint.ip }}
+    ip = {{ .Values.kubernetes.ingressEndpoint.ip | quote }}
+      {{- end}}
+      {{- if .Values.kubernetes.ingressEndpoint.publishedService }}
+    publishedService = {{ .Values.kubernetes.ingressEndpoint.publishedService | quote }}
+      {{- end}}
+    {{- else }}
+    [kubernetes.ingressEndpoint]
+      publishedService = "{{ .Release.Namespace }}/{{ template "traefik.fullname" . }}"
```

Comment:
Not a fan of this personally. I think the note in the readme should be sufficient to prompt people what to use

Reply:
Traefik cannot function if it's empty or not defined.

Reply:
Oh, I see. You want to disable this function.
Is it good for you if I create a flag like this?
```
    {{- else if .Values.kubernetes.ingressEndpoint.useDefaultServiceName }}
    [kubernetes.ingressEndpoint]
      publishedService = "{{ .Release.Namespace }}/{{ template "traefik.fullname" . }}"
    {{- end}}
```